### PR TITLE
Added explainer draft.

### PR DESCRIPTION
**Pull request by shangl**

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Introduction
+`getAllScreensMedia` is an API that allows clients to capture all monitors
+attached to a device at once without user interaction. It is only available in
+managed sessions for allow-listed web apps and it is a requirement
+that the user must be informed about capturing at all times. The usage
+indicator cannot be prevented by the web app.
+
+## Motivation
+Partners have asked for such an API in order to meet legal and internal
```
Comment (eladalon1983) [inline explainer.md]: A reader outside of CrOS would potentially be confused about who these partners are. I suggest employing a more neutral POV, speaking about how Web developers have certain needs.

File: explainer.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+compliance requirements. Interest for such an API was
+[publicly expressed](https://github.com/w3c/mediacapture-screen-share/issues/204).
+
+## Use cases
+* In the financial industry a consultant may provide financial advice digitally
+and a complete documentation of the information may be required by law in some
+jurisdictions.
+* Internet usage in prisons may require full traceability to allow convicts to
+access the internet.
+* Contact centers may require full documentation of provided information for
```
Comment (eladalon1983) [inline explainer.md]: nit: Could we put this one first?

File: explainer.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* Constraints cannot be passed in the `getAllScreensMedia` call. Constraints may be
+different depending on the monitor and information determining the desired
+constraints (resolution, size, …) is likely only available after the monitor
+was captured. Constraints can be applied on the returned `MediaStreamTrack`
+with `applyConstraints`.
+* `getAllScreensMedia` will only be available for web apps allowlisted by a
+policy.
+* Usage indicators must be shown to the user at all times.
+
+`getAllScreensMedia` will use `ScreenCaptureMediaStreamTrack`in the
```
Comment (eladalon1983) [inline explainer.md]: Missing space between `ScreenCaptureMediaStreamTrack` and "in".
Could you also explain that SCMST is a subclass of MST?

File: explainer.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+returned `MediaStream` which provides access to monitor details analogous
+to the
+[getScreenDetails API](https://developer.chrome.com/articles/multi-screen-window-placement/#the-getscreendetails()-method).
+
+```cs
```
Comment (eladalon1983) [inline explainer.md]: Should this be \`\`\`idl? What's \`\`\`cs?

File: explainer.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+partial interface MediaDevices {
+ Promise<sequence<MediaStream>> getAllScreensMedia();
+}
+```
+
+```cs
```
Comment (eladalon1983) [inline explainer.md]: Ditto.

**Review by eladalon1983 — APPROVED** [review]

File: explainer.md (hunk 4 above)
Comment (shangl) [reply explainer.md]: cs == c#. to my knowledge there is no idl highlighting, and cs correctly highlights at least parts. asking around on what to use here i got recommended to use cs. replaced it by idl now anyways.

**Pull request merged**